<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Safe</class>
 <widget class="QWidget" name="Safe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>115</width>
    <height>235</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="value" stdset="0">
      <double>88888.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QToolButton" name="btn_0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="btn_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="btn_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QToolButton" name="btn_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QToolButton" name="btn_1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="btn_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="btn_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="btn_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btn_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="btn_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="btn_clear">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btn_back">
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_0</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
